cmd/kubeadm/app/cmd/token.go

```diff
@@ -85,20 +93,26 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 		Use:   "list",
 		Short: "List bootstrap tokens on the server.",
 		Run: func(tokenCmd *cobra.Command, args []string) {
-			err := RunListTokens(out, errW, tokenCmd)
+			client, err := kubemaster.CreateClientFromFile(getKubeConfigPath())
+			kubeadmutil.CheckErr(err)
```

Comment:
Regardless, it is a common pattern in many of the utilities in k8s.  I see this in `kubectl` and `kubefed` also.  If we want to change it it should probably be done in a different PR.